cmd/manager/main.go

```diff
@@ -56,11 +56,15 @@ func main() {
 	var registryPort int
 	var registryProtocol string
 	var registryURL string
-	var requeueInterval time.Duration
+	var registryResyncInterval time.Duration
 	var webhookPort int
 	var enforceFirstBoot bool
+	var resyncInterval time.Duration
 
-	flag.DurationVar(&requeueInterval, "requeue-interval", 10*time.Second, "Reconciler requeue interval.")
+	flag.DurationVar(&registryResyncInterval, "registry-resync-interval", 10*time.Second,
+		"Defines the interval at which the registry is polled for new server information.")
+	flag.DurationVar(&resyncInterval, "resync-interval", 30*time.Second,
```

Comment:
Optional: If it applies only to the `Server` CRs, `server-resync-interval` or `server-requeue-interval` could be helpful.